utils/runtime-fuzzer/src/lib.rs

```diff
@@ -106,10 +119,23 @@ fn run_impl(data: &[u8]) -> Result<sp_io::TestExternalities> {
     Ok(test_ext)
 }
 
-fn execute_gear_call(sender: AccountId, call: GearCall) -> DispatchResultWithPostInfo {
+fn execute_gear_call(
+    sender: AccountId,
+    call: GearCall,
+    fuzzing_config: &FuzzingConfig,
+) -> DispatchResultWithPostInfo {
+    let allowed_to_spend_value = if fuzzing_config.allow_exceed_sender_balance {
+        u128::MAX
```

Comment:
I think `allow_exceed_sender_balance` should just "allow" this, but not exceed the sender's balance every time. Maybe make it random?